inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_check.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gna/gna_config.hpp>
+
+class GnaLayerTestCheck : virtual public LayerTestsUtils::LayerTestsCommon {
+protected:
+    bool skipTest = true;
```

Comment:
Why is skipTest field needed? Why can't SkipTestCheck() just return bool value?

Reply:
This is because GTEST_SKIP() forces a different return type and aborts the execution of the current function. But thanks to your comment I spotted a bug which should be now fixed.